Makefile

```diff
@@ -50,25 +50,28 @@ binary: build
 	$(DOCKER_RUN_DOCKER) hack/make.sh binary
 
 cross: build
-	$(DOCKER_RUN_DOCKER) hack/make.sh binary cross
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary binary cross
 
 deb: build
-	$(DOCKER_RUN_DOCKER) hack/make.sh binary build-deb
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary build-deb
+
+dynbinary: build
+	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary
```

Comment:
I've seen this introduced here as well https://github.com/diogomonica/docker/commit/50852cdc4d7ecfdab795018309aca783880ec12d#diff-b67911656ef5d18c4ae36cb6741b7965R51
PR is https://github.com/docker/docker/pull/17937